### ISSUE TITLE
create view relation has extra quotes (") and schema name in 10.1

### ISSUE THREAD

**Issue by huisben** [fishtown-analytics/dbt#792]
I noticed once I updated to 10.1, our view materializations error out due to an extra wrapping of "" around the table name and the schema is repeated

I haven't noticed this in any of our ephemeral or table models.

<img width="707" alt="image" src="https://user-images.githubusercontent.com/5601774/41324238-62111598-6e68-11e8-9923-71b58bf9a91f.png">

Ben

**Comment by drewbanin** [comment]
Hey @huisben - thanks for the report. Couple of questions:

What kind of warehouse are you using? 
Did you configure [quoting](https://docs.getdbt.com/docs/configuring-quoting) in your `dbt_project.yml` file?

This definitely doesn't look right, I'll take a look.

**Comment by huisben** [comment]
hey @drewbanin thanks for the tip on quoting. Makes a lot of sense. 

I've added the explicit quoting in and getting similar errors.  I tried for all permutations. With both off, there are no quotes but database complains it cannot find the table. With one or the other on it will throw double quotes around whichever one I toggle. 

With both on it looks now like this `create view "dbt_ben".""dbt_ben"."basic_accounts__dbt_tmp"" `

we're on redshift.

**Comment by huisben** [comment]
it actually seems like it's just writing the schema twice, rather than a quoting problem

**Comment by huisben** [comment]
thanks to Drew, this "issue" (it actually was not an issue with dbt, at all) is RESOLVED.

for future reference of anyone who may see this, we had created a macro in our `dbt/macros` folder to do late binding views on our Redshift cluster. 

Since dbt has natively supported this, we did not update it. It began manifesting itself as an issue with 10.1 because of the new Relation model (https://github.com/fishtown-analytics/dbt/blob/master/dbt/adapters/default/relation.py) which itself contains both a `schema` and `identifier` component. Because our legacy code appended `schema` and `identifier` is now `Relation`, it was causing this error. 

Thanks Drew and the dbt community!!
